### PR TITLE
Fix/volumetemplate

### PR DESCRIPTION
**Pull request by WafflesVonMaple**
FIx volume templates to allow override for cloud storageclass

### CODE REVIEW

File: helm/sequencer/values.yaml [hunk 1]
```diff
@@ -45,3 +45,6 @@ storage:
     persistentVolumeName: "sequencer-shared-storage"
     storageClassName: "sequencer-shared-storage-local"
     path: "/data/sequencer-data"
+
+# Allows to override storage class for when the storage class already exists, like in GKE.
+sharedStorageClassOverride: ""
```
Comment (steezeburger) [inline helm/sequencer/values.yaml:50]: nit newline

File: helm/rollup/templates/volumes.yaml [hunk 2]
```diff
@@ -32,18 +32,44 @@ spec:
...
+  storageClassName: "{{ $.Values.rollupName }}-{{ $value.storageClassName }}"
+  {{- end }}
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: {{ $value.size }}
+
+---
+{{ end }}
```
Comment (steezeburger) [inline helm/rollup/templates/volumes.yaml:75]: nit newline

**Comment by WafflesVonMaple** [comment]
setting storage class directly in each chart is just going to be messy and hard for folks to take beyond local. If no-provision storage class is really necessary for local, recommend just putting in the local cluster spec 

`apiVersion: storage.k8s.io/v1
kind: StorageClass
metadata:
  name: default
provisioner: kubernetes.io/no-provisioner`

**Review by steezeburger — APPROVED** [review]
Approved but can you add the missing newlines from my other comments?

**Comment by WafflesVonMaple** [comment]
@joroshiba can we also merge this in before it becomes stale

File: helm/sequencer/values.yaml [hunk 3]
```diff
@@ -43,5 +43,5 @@ storage:
     type: "local"
     size: "5Gi"
     persistentVolumeName: "sequencer-shared-storage"
-    storageClassName: "sequencer-shared-storage-local"
     path: "/data/sequencer-data"
+
```
Comment (joroshiba) [inline helm/sequencer/values.yaml:47]: nit: I think there is some white space on this new line

**Review by joroshiba — APPROVED** [review]
LGTM, 1 nit

File: helm/rollup/templates/storageclasses.yaml
Comment (joroshiba) [inline helm/rollup/templates/storageclasses.yaml:1]: I think we actually still want this file, for testing storage running locally?

This is really just about making sure if running locally and using some persistent storage things don't go monumentally sideways, and can be useful when testing things like restart logic.

**Pull request merged**